### ISSUE TITLE
zfs_tests missing commands.cfg CentOS 7

### ISSUE THREAD

**Issue by memnochproxy** [zfsonlinux/zfs#6970]
### System information

Type                                | Version/Name
  ---                                  |     --- 
Distribution Name       | CentOS 7
Distribution Version    | CentOS Linux release 7.4.1708 (Core)
Linux Kernel                 | 3.10.0-693.2.2.el7.x86_64
Architecture                 | x86_64
ZFS Version                  | 0.7.4
SPL Version                  | 0.7.4

### Describe the problem you're observing
Unable to build zfs-0.7.4

### Describe how to reproduce the problem
cd spl
make distclean && ./autogen.sh && ./configure && make -j5 && sudo make install
cd ../zfs
make distclean && ./autogen.sh && ./configure && make -j5 && sudo make install

### Include any warning/errors/backtraces from the system logs

```
Making all in include
make[4]: Entering directory `/home/jreynolds/local/zfs/tests/zfs-tests/include'
make[4]: *** No rule to make target `commands.cfg', needed by `all-am'.  Stop.
make[4]: Leaving directory `/home/jreynolds/local/zfs/tests/zfs-tests/include'
make[3]: *** [all-recursive] Error 1
make[3]: Leaving directory `/home/jreynolds/local/zfs/tests/zfs-tests'
make[2]: *** [all-recursive] Error 1
make[2]: Leaving directory `/home/jreynolds/local/zfs/tests'
make[1]: *** [all-recursive] Error 1
make[1]: Leaving directory `/home/jreynolds/local/zfs'
make: *** [all] Error 2
```

**Comment by bmerchant** [comment]
It seems `make distclean` removes a number of files it probably shouldn't. Below is the `ls -R` diff from before and after `./autogen.sh && ./configure && make distclean` on a freshly untarred zfs-0.7.4 release source.

```
lib/libzfs/libzfs.pc
lib/libzfs/libzfs_core.pc
scripts/common.sh
tests/zfs-tests/include/commands.cfg
tests/zfs-tests/tests/functional/clean_mirror/default.cfg
tests/zfs-tests/tests/functional/scrub_mirror/default.cfg
tests/zfs-tests/tests/functional/checksum/default.cfg
tests/zfs-tests/include/default.cfg
tests/zfs-tests/include/libtest.shlib
tests/zfs-tests/include/math.shlib
tests/zfs-tests/include/properties.shlib
tests/zfs-tests/include/zpool_script.shlib
```

**Comment by bmerchant** [comment]
Brain fart for not searching first. This was fixed in #6636.
